library/CMakeLists.txt

```diff
@@ -81,7 +81,7 @@ set( HIPBLAS_CONFIG_DIR "\${CPACK_PACKAGING_INSTALL_PREFIX}/${CMAKE_INSTALL_LIBD
 rocm_create_package(
     NAME ${package_name}
     DESCRIPTION "Radeon Open Compute BLAS marshalling library"
-    MAINTAINER "Kent Knox <kent.knox@amd.com>"
+    MAINTAINER "rocblas-maintainer@amd.com"
     LDCONFIG
```

Comment:
can this change to    hipblas-maintainer@amd.com

Reply:
@saadrahim could you add  hipblas-maintainer@amd.com? thanks